website/docs/file_sizing.md

```diff
@@ -3,51 +3,76 @@ title: "File Sizing"
-same configurations as above for the COPY_ON_WRITE case applies.
-
-NOTE : In either case, small files will be auto sized only if there is no PENDING compaction or associated log file for 
-that particular file slice. For example, for case 1: If you had a log file and a compaction C1 was scheduled to convert 
-that log file to parquet, no more inserts can go into that log file. For case 2: If you had a parquet file and an update 
-ended up creating an associated delta log file, no more inserts can go into that parquet file. Only after the compaction 
-has been performed and there are NO log files associated with the base parquet file, can new inserts be sent to auto size that parquet file.
-
-## Auto-Size With Clustering
-**[Clustering](/docs/next/clustering)** is a feature in Hudi to group 
-small files into larger ones either synchronously or asynchronously. Since first solution of auto-sizing small files has 
-a tradeoff on ingestion speed (since the small files are sized during ingestion), if your use-case is very sensitive to 
-ingestion latency where you don't want to compromise on ingestion speed which may end up creating a lot of small files, 
-clustering comes to the rescue. Clustering can be scheduled through the ingestion job and an asynchronus job can stitch 
-small files together in the background to generate larger files. NOTE that during this, ingestion can continue to run concurrently.
-
-*Please note that Hudi always creates immutable files on disk. To be able to do auto-sizing or clustering, Hudi will 
-always create a newer version of the smaller file, resulting in 2 versions of the same file. 
-The [cleaner service](/docs/next/hoodie_cleaner) will later kick in and delete the older version small file and keep the latest one.*
\ No newline at end of file
+A fundamental problem when writing data to a source is having a lot of small files. This is also known as a small file problem. If you don’t size the files appropriately, you can slow down the query performance and work with stale analytics. Some of the issues you may encounter with small files include the following:
+- **Reads slow down**: You’ll have to scan through many small files to retrieve data for a query. It’s a very inefficient way of accessing and utilizing the data.
+
+- **Processes slow down**: You can slow down your i.e., Spark or Hive jobs; the more files you have, the more tasks you create for reading each file.
+
+- **Storage increases**: When working with a lot of data, you can be inefficient in using your storage. For example, many small files can have a lower compression ratio, leading to more data on storage. If you’re indexing the data, that also takes up more storage space inside the Parquet files. If you’re working with a small amount of data, you might not see a significant impact with storage. However, when dealing with petabyte and exabyte data, you’ll need to be efficient in managing storage resources. 
```

Comment:
Should we change heading from "Storage increases" -> "Inefficient use of storage" ? I think, that is more appropriate ?

Reply:
ahh -- I was trying to standardize the bullets so it's all consistent.

Reply:
@bhasudha @yihua -wdyt: **Storage use inefficiencies**